cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,34 @@
 /// <reference types="cypress" />
 
-describe('Sign In page', () => {
+describe('Site page', () => {
+  beforeEach(() => {
+    cy.visit('https://react-redux.realworld.io');
+  });
+
+  it('should open main page', () => {
+    cy.get('h1').should('contain.text', 'conduit');
+  });
+
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io/#/login?_k=y0qwae');
+  });
+
+  it('should click on [Sign in] button', () => {
+    cy.contains('a', 'Sign in').should('exist').click();
+    cy.get('h1').should('contain.text', 'Sign In');
+  });
+
+  it('should fill in email and password fields with valid data', () => {
```

Comment:
this test will fail because the link in "beforeEach" function redirects to main page, not Sign In , so cypress cant find Email field

Reply:
fixed